source/run-command.txt

```diff
@@ -0,0 +1,182 @@
+.. _c-run-command:
+
+======================
+Run a Database Command
+======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :description: Learn how to use the C Driver to run a database command.
+   :keywords: command, server, usage, code example
+
+Overview
+--------
+
+In this guide, you can learn how to run a database command with the
+{+driver-short+}. You can use database commands to perform a variety of
+administrative and diagnostic tasks, such as fetching server statistics,
+initializing a replica set, or running an aggregation pipeline.
+
+.. important:: Prefer Driver Methods to Database Commands
+
+   The driver provides wrapper methods for many database commands. We recommend
+   using driver methods instead of executing database commands when possible.
```

Comment:
We refer to them as "functions" rather than "methods" in the C driver. Applies to all instances
```suggestion
   using driver functions instead of executing database commands when possible.
```